.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -4,14 +4,20 @@ about: Suggest an idea for this project
 
 ---
 
-## Description
+## Problem
 
-A short summary of the idea.
+What is the user, security, system, or project sustainability need this Issue was created to address? Is there evidence to cite in feedback from users, contributors, or instance owners? Is there evidence to cite in a report from a domain expert, in ux or dev best practices, or in user research findings? Or, is this something you simply feel compelled to address?
```

Comment:
This is overly verbose, to the point of being off-putting, IMO. It combines the former description section with the former user research section in an unclear way, and asks folks to justify their request to a level of detail that verges on hostile.